examples/ReferenceTestV2/reference_consumer_v2.py

```diff
@@ -0,0 +1,601 @@
+    generated_family_names = []
+    if len(patient_context_descriptors) == 0:
+        log_result(False, results, step, info, extra_info='no PatientContextDescriptor')
+    else:
+        try:
+            for i, p in enumerate(patient_context_descriptors):
+                pat = client.context_service_client.mk_proposed_context_object(p.Handle)
+                pat.CoreData.Familyname = uuid.uuid4().hex
+                pat.ContextAssociation = pm_types.ContextAssociation.ASSOCIATED
+                generated_family_names.append(pat.CoreData.Familyname)
+                client.context_service_client.set_context_state(set_context_state_handle, [pat])
+            time.sleep(1)  # allow update notification to arrive
+            patients = mdib.context_states.NODETYPE.get(pm_qnames.PatientContextState, [])
+            if len(patients) == 0:
+                log_result(False, results, step, info, extra_info='no patients found')
+            else:
+                all_ok = True
+                for patient in patients:
+                    if patient.CoreData.Familyname in generated_family_names:
+                        if patient.ContextAssociation != pm_types.ContextAssociation.ASSOCIATED:
+                            log_result(False, results, step, info,
+                                       extra_info=f'new patient {patient.CoreData.Familyname} is {patient.ContextAssociation}')
+                            all_ok = False
+                    else:
+                        if patient.ContextAssociation == pm_types.ContextAssociation.ASSOCIATED:
+                            log_result(False, results, step, info,
+                                       extra_info=f'old patient {patient.CoreData.Familyname} is {patient.ContextAssociation}')
+                            all_ok = False
+                log_result(all_ok, results, step, info)
+        except Exception as ex:
+            print(traceback.format_exc())
+            log_result(False, results, step, info, ex)
+
+    step = '6c'
+    info = 'SetValue: Immediately answers with "finished"'
+    print(step, info)
+    subscriptions = client.subscription_mgr.subscriptions.values()
+    operation_invoked_subscriptions = [subscr for subscr in subscriptions
+                                       if 'OperationInvokedReport' in subscr.short_filter_string]
+    if len(operation_invoked_subscriptions) == 0:
+        log_result(False, results, step, info, 'OperationInvokedReport not subscribed, cannot test')
+    elif len(operation_invoked_subscriptions) > 1:
+        log_result(False, results, step, info,
+                   f'found {len(operation_invoked_subscriptions)} OperationInvokedReport subscribed, cannot test')
+    else:
+        try:
+            operations = client.mdib.descriptions.NODETYPE.get(pm_qnames.SetValueOperationDescriptor, [])
+            my_ops = [op for op in operations if op.Type.Code == "67108888"]

```

Comment:
please extract `67108888` to a variable and give it a proper name. Also, codes should not be compared like this